### ISSUE TITLE
Slider datetime return NaN

### ISSUE THREAD

**Issue by weibangtuo** [IonDen/ion.rangeSlider#358]
``` javascript
$("#range").ionRangeSlider({
    min: 0,
    max: +new Date(),
    from: 0,
    prettify_enabled: true,
    prettify: function (val) { // val is NaN
        //return formatterDate(val);
    }
});
```

May `max - min` is too large. if  min and max diff in days, it's work well. Or use like under code

``` javascript
// it's work!
$("#range").ionRangeSlider({
    min: 0,
    max: ~~(new Date() / 1000), 
    from: 0,
    prettify_enabled: true,
    prettify: function (val) { 
        return formatterDate(val * 1000);
    }
});
```

**Comment by IonDen** [comment]
Hello, thanks for finding it. I will investigate.

**Comment by IonDen** [comment]
Should be fixed in 2.1.5
